utils/frontend_utils.py

```
@@ -385,19 +398,41 @@ def find_model_directory(model_name: str):
-    
-    # 首先尝试在用户文档目录
+
+    # 获取 CFA 场景下的可读 live2d 目录（可能为 None）
+    readable_live2d = None
+    try:
+        config_mgr = get_config_manager()
+        readable_live2d = config_mgr.readable_live2d_dir
+    except Exception:
+        pass
+
+    # 首先尝试在用户文档目录（可写回退路径，如 AppData）
```

Ruff (0.15.5)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)